spiped/README

```diff
@@ -21,9 +21,12 @@ Options:
 		[ipv6::addr]:port
 	Note that hostnames are resolved when spiped is launched and are not
 	re-resolved later; thus if DNS entries change spiped will continue to
-	connect to the expired address.
+	accept at the expired address.
     -t <target socket>
-	Address to which spiped should connect.
+	Address to which spiped should connect. The accepted formats are the
```

Comment:
I did not adapt the docs for *spipe* yet. I think it might be best to copy lines 17 to 21 of this file into the docs of *spipe*. What do you think?